feat: add Sparky

mixcoac opened the pull request:
Hey there,

I designed/developed this pretty simple theme called Sparky; its based on Enter Shikari's *The Spark* and *Nothing is True & Everything is Possible*.

Its a little similar to Solarized but I guess it has its own spin of post-hardcore British rock.

![image](https://github.com/Gogh-Co/Gogh/assets/35469663/4c1620d1-7315-4ccb-b678-4e9c8e9c7997)

Mgldvd commented:
Hello, please keep the file format with comments, also remove the 3 dashes at the end.
Thank you!

mixcoac commented:
There, reformatted the YML to have comments and removed the 3 dashes; also switched to apostrophes just in case.

PD: This also made me realise that I had incorrectly filed my scheme (set my darks as brights) so I guess this mistake also came with a good ending heh.
